src/node-lib/getServerConfig.ts

```diff
@@ -246,6 +246,20 @@ const envVars = satisfies<Record<string, EnvVar>>()({
     availableInBrowser: false,
     default: null,
   },
+  openAiApiKey: {
+    value: process.env.OPENAI_API_KEY,
+    envName: "OPENAI_API_KEY",
+    required: true,
+    availableInBrowser: false,
+    default: null,
+  },
+  searchEnabledInProd: {
+    value: process.env.SEARCH_ENABLED_IN_PROD,
+    envName: "SEARCH_ENABLED_IN_PROD",
+    required: true,
```

Comment:
Is this is required in all environments, then it might make sense for it just to be `AI_SEARCH_ENABLED` and then it can be set to true,, true, and false in dev, staging, and prod environments. I've pinged you a message on slack with more detail

I don't think it's that important and doesn't have to block this PR